x-pack/plugins/ingest_manager/public/applications/ingest_manager/components/enrollment_instructions/manual/index.tsx

```diff
@@ -4,33 +4,53 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { EuiText, EuiSpacer } from '@elastic/eui';
 import React from 'react';
+import { EuiText, EuiSpacer, EuiCode, EuiCodeBlock, EuiCopy, EuiButton } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EnrollmentAPIKey } from '../../../types';
 
-export const ManualInstructions: React.FunctionComponent = () => {
+interface Props {
+  kibanaUrl: string;
+  apiKey: EnrollmentAPIKey;
+  kibanaCASha256?: string;
+}
+
+export const ManualInstructions: React.FunctionComponent<Props> = ({
+  kibanaUrl,
+  apiKey,
+  kibanaCASha256,
+}) => {
+  const command = `
+./elastic-agent enroll ${kibanaUrl} ${apiKey.api_key}${
+    kibanaCASha256 ? ` --ca_sha256=${kibanaCASha256}` : ''
+  }
+./elastic-agent run`;
```

Comment:
does this need `sudo` or a note about running these commands as root?

Reply:
I think you should not run these as root

Reply:
I think with recent changes @michalpristas said that it needs to be run as root because agent performs chown, chmod, etc actions

Reply:
Oh ok I was not aware of that, if @ph or @michalpristas can confirm the agent need to be runned as root I can add a sudo here

Reply:
atm we made changes so it can run without sudo. these chown chmod operations were performed at /var/run level before. we changed that so you should not need root.
maybe a thing to point out is that user which runs the agent will be used to run beats as well.